tools/compress/main.cpp

```diff
@@ -252,6 +252,11 @@ int main(int argc, const char** argv)
             return_code = -1;
         }
     }
+    else
+    {
+        GFXRECON_WRITE_CONSOLE("CompressionCoverter could not be initialized.");
+        exit(-1);
```

Comment:
Would it be better/more consistent/allow for proper cleanup to have `return_code = -1;` like in the failure case above?

Reply:
That's a subtle one - I was going to do `return_code` but `exit(-1)` is already used in 3 other places in `main` so I judged `exit` to be more consistent.  I'll dwell on it today; it's slightly different in that the `exit` is in the `else` very far away from the function that precipitated it.

Reply:
I surveyed the other tools; they use `exit` for failure handling except the last two failure possibilities in `optimize`.  So I went the other direction, using exit() for failures through the code in `compress`.  Does that work for you @lunarpapillo ?

Reply:
Additionally I realized I didn't close the Log on exit while making this change, so fixed that.

Reply:
@lunarpapillo